osu.Game/Overlays/Settings/Sections/Maintenance/GeneralSettings.cs

```diff
@@ -58,6 +59,19 @@ private void load(BeatmapManager beatmaps, ScoreManager scores, SkinManager skin
                 }
             });
 
+            Add(deleteBeatmapVideosButton = new DangerousSettingsButton
+            {
+                Text = MaintenanceSettingsStrings.DeleteAllBeatmapVideos,
+                Action = () =>
+                {
+                    dialogOverlay?.Push(new MassVideoDeleteConfirmationDialog(() =>
+                    {
+                        deleteBeatmapVideosButton.Enabled.Value = false;
+                        Task.Run(() => beatmaps.DeleteVideos()).ContinueWith(t => Schedule(() => deleteBeatmapVideosButton.Enabled.Value = true));
```

Comment:
One of the failing code quality checks mentions converting this line to a method group - however I just made it similar to the `beatmaps.Delete()` call right above it. 

Okay to just fix the missing line break one in `MassVideoDeleteConfirmationDialog`?